process/interface.go

```diff
@@ -1125,6 +1126,7 @@ type WhiteListHandler interface {
 
 // InterceptedDebugger defines an interface for debugging the intercepted data
 type InterceptedDebugger interface {
+	LogReceivedData(data InterceptedData, msg p2p2.MessageP2P, fromConnectedPeer core.PeerID)
```

Comment:
should try to avoid importing communication-go here, but instead make an alias inside the p2p package of mx-chain-go

Reply:
used the already existing alias from mx-chain-go p2p package